test/suites/smoke/test-staking-rewards.ts

```diff
@@ -498,9 +498,7 @@ describeSuite({
       // calculate reward amounts
       const parachainBondInfo = await apiAtPriorRewarded.query.parachainStaking.parachainBondInfo();
       const parachainBondPercent = new Percent(parachainBondInfo.percent);
-      const totalStaked = await apiAtPriorRewarded.query.parachainStaking.staked(
-        originalRoundNumber
-      );
+      const totalStaked = await apiAtPriorRewarded.query.parachainStaking.totalSelected();
```

Comment:
This change is wrong, `totalSelected` is the number of selected collators, and because it's alwauys lower than the min inflation expectation it break the smoke test on moonbeam because they always apply `range.min` instead of `range.ideal`